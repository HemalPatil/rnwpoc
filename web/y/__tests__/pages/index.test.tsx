import { render } from '@testing-library/react'

import HomePage from '../../pages'

describe('<HomePage />', () => {
  it('Should match snapshot', () => {
    const { container } = render(<HomePage />)
    expect(container).toMatchSnapshot()
  })
})
