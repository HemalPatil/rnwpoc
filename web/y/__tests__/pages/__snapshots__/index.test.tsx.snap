// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> Should match snapshot 1`] = `
<div>
  <div>
    rnwpoc-web-y
  </div>
  <div
    class="css-view-175oi2r r-cursor-1loqt21 r-touchAction-1otgn73"
    tabindex="0"
  >
    <div
      class="css-text-146c3p1"
      dir="auto"
    >
      This is an RNW button
    </div>
  </div>
  <button>
    This is an HTML5 button
  </button>
</div>
`;
